### DISCUSSION TITLE
Lay Z Spa Timer

### DISCUSSION THREAD

**Discussion by Mark18PLL** [visualapproach/WiFi-remote-for-Bestway-Lay-Z-SPA#801] [Q&A]
I have not installed the Wifi module yet but have a question about timers.  I would like to be able to set the heater so it only comes on between say 9am and 5pm and also have a manual override that wont wipe the timer settings, is this possible? I dont really want to use HA or similar, I just find them painful and very time consuming if you are not completely upto speed with them and would like a simple solution. 
Thanks

**Comment by visualapproach** [comment]
Yes you can do that.

**Reply by Mark18PLL** [reply]
Thats great, can you give me an idea how to do it please or point me in the right direction?  Thanks Mark

**Reply by visualapproach** [reply]
The manual will give you a hint. If something needs clarification I'd be happy to help

**Reply by Mark18PLL** [reply]
I have looked at the spa config a few times but I cant seem to workout how I would set up a schedule that it follows daily but I can override.

**Reply by visualapproach** [reply]
You set the action and time and repeat interval. So for example 
Set heater on
Value: 1 (=on)
Time: 08:00
Interval: 86400s (a day)

Set pump : 0 (turns off pump and heater)
Time: 16:00
Interval: 86400

These actions will trigger every day and you can 'override' if you want to call it that, by pressing buttons on the display or in the UI.

**Reply by Mark18PLL** [reply]
Thanks, I understand.  Last question hopefully lol, if I wanted the pump to stay on when the heater is turned off, is this possible?

**Reply by visualapproach** [reply]
Yes, just turn off the heater

**Reply by Mark18PLL** [reply]
Ah so in the settings you posted above, on the second one (set pump) if I change that to set heater to off, it will leave the pump on?
Thanks 
Think you will be due a coffee off me soon lol.

**Reply by visualapproach** [reply]
That's right :)
